spec/support/platform_helpers.rb

```diff
@@ -243,10 +243,10 @@ def ifconfig?
 
 def choco_installed?
   result = ShellHelpers.powershell_exec("choco --version")
-  result.error?
+  !result.error?
 end
 
 def pwsh_installed?
   result = ShellHelpers.powershell_exec("pwsh.exe --version")
-  result.error?
+  !result.error?
```

Comment:
This seems like kind of a big deal, are these just broken right now?

Reply:
yeah. this is all a bit rough right now. and ^^^ broke alot of stuff :)